src/components/Paginate.tsx

```diff
@@ -98,7 +98,7 @@ export default function Paginate({
     for (
       let index = numberOfPages - NUMBER_OF_ALWAYS_PRESENT_PAGE_BUTTONS - 1;
       index < numberOfPages; // We always show the page before the current
-      index++ // one; this is why we substract one from the starting index.
```

Comment:
The breaks in the line spacing of the for loop make it very difficult to read. Its pretty uncommon for code comments to not be on their own line. A code comment above a line is always understood to refer to the line below it.

Reply:
I also think it would be good to perform similar changes to the rest of this file to improve its readability.

At a high level, its important that all the people contributing to a single codebase converge on similar coding conventions. Otherwise the codebase becomes inconsistent and hard to read. Thinking of this from the perspective of a new reader, they might be wondering why comments in a certain place are inline with code while other comments are above the line of code that they refer to.

Reply:
I think converging on similar coding conventions make a lot of sense. c0373da, which is the commit I just pushed, makes comments only describe their own line of code.

Thanks for the feedback. Also, happy New Year!

Reply:
My bad, I should have added an example to clarify. By new line I mean:

```ts
// bad
const foo = 123 // initialize foo
```

```ts
// good
// initialize foo
const foo = 123
```

Reply:
Ah, I see. I reverted the second change I made, and the commit after that reversion, 45c4064, makes all comments go on their own line of code.